grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy

```diff
@@ -18,6 +18,7 @@ import org.pih.warehouse.core.Location
 import org.pih.warehouse.core.Person
 import org.pih.warehouse.core.RoleType
 import org.pih.warehouse.core.User
+import org.pih.warehouse.exporter.PickListItemExcelExporter
```

Comment:
I get that this might be confusing and not seem all that important. But when we're talking about a single resource, we should make sure not to over-capitalize. This goes for other WMS-specific terms like putaway as well. In case it's not clear, it should be Putaway, not PutAway. We should be consistent across the application but I'm not sure how to make it more clear and intuitive. Any thoughts on how we can do that? Maybe a glossary?